site/en/docs/extensions/mv3/hosting/index.md

```diff
@@ -2,68 +2,50 @@
 layout: "layouts/doc-post.njk"
 title: "Chrome Web Store"
```

Comment:
We should rename this file to something else as it is only tangentially related to "Chrome Web Store". Maybe "Extension hosting"?

Reply:
@dotproto Any thoughts on the description?